packages/tpetra/core/src/Tpetra_CrsGraph_decl.hpp

```diff
@@ -590,9 +590,32 @@ namespace Tpetra {
       return copier_type::clone (*this, node2, params);
     }
 
+
     //! Destructor.
     virtual ~CrsGraph () = default;
 
+
+    /// \brief Swaps the data from *this with the data and maps from graph
+    ///
+    /// \param graph [in/out] a crsGraph
+    void swap(CrsGraph<LocalOrdinal, GlobalOrdinal, Node> & graph);
+
+
+    /// \brief True if and only if \c CrsGraph is identical to this CrsGraph
```

Comment:
In a later PR (or this one, if you plan on making changes), please add the following text to the documentation:
```
/// \warning THIS FUNCTION IS NOT FOR USERS.  WE MAKE NO PROMISES OF BACKWARDS COMPATIBILITY.  THIS FUNCTION MAY DISAPPEAR OR CHANGE AT ANY TIME.  WE ARE NOT RESPONSIBLE FOR WHATEVER HAPPENS IF USERS SHOULD ATTEMPT TO CALL THIS FUNCTION.
```
Thanks!